### PR TITLE
Updated Wiki, for real this time

### PR DESCRIPTION
**Pull request by Xiribidus**

### CODE REVIEW

File: wiki/rats-mischief.md [hunk 1]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
 <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 [![img](https://img.shields.io/discord/292744693803122688?color=informational&label=Ladysnake&logo=Discord)](https://ladysnake.glitch.me)[![img](http://cf.way2muchnoise.eu/full_rats-rats_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](http://cf.way2muchnoise.eu/versions/minecraft_rats-rats_latest.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](https://img.shields.io/github/last-commit/ladysnake/rats-mischief)](https://github.com/ladysnake/rats-mischief/commits/main)
 
+
+# Rat's Mischief
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]: You can remove the title, it's added automatically on the website
```suggestion
```

File: wiki/rats-mischief.md [hunk 2]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+[![img](https://img.shields.io/discord/292744693803122688?color=informational&label=Ladysnake&logo=Discord)](https://ladysnake.glitch.me)[![img](http://cf.way2muchnoise.eu/full_rats-mischief_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](http://cf.way2muchnoise.eu/versions/minecraft_rats-mischief_latest.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](https://img.shields.io/github/last-commit/ladysnake/rats-mischief)](https://github.com/ladysnake/rats-mischief/commits/main)
+
 **Rat's Mischief** (also known as "Rat's Rats" or just "Mischief") is a Minecraft mod introducing a new mob, the rat, as well as mechanics revolving around these new possible pets.
 
+[![Showcase Video](https://img.youtube.com/vi/CVZfsPM8Mm4/0.jpg)](https://www.youtube.com/watch?v=CVZfsPM8Mm4)
+
+# Wiki
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]: There should only be 1 first-level title per page, so all those titles should be one level lower
```suggestion
## Features
```

File: wiki/rats-mischief.md [hunk 3]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+## Rats
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
### Rats
```

File: wiki/rats-mischief.md [hunk 4]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+Rats can be found in previously inhabited villages that are now abandoned. Once a natural village no longer has tenants, rats will start spawning if a player is near that empty village and the village still has beds.
+
+### Behaviours
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Behaviours
```

File: wiki/rats-mischief.md [hunk 5]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+Rats will behave in a very similar manner to wolves, attacking what the player targets or what harms the player or themselves. They have 8 health points (4 hearts) and deal one point of damage but ignore invulnerability ticks, making a mischief stronger the bigger it becomes.
+Rats take fall damage starting from 15 blocks, can breathe five times longer than normal creatures, ignore damage from cacti and berry bushes and can be tamed with any food, the better the food though, the higher the chance of success. Once tamed you will be able to sit them down and or let them follow you. If they see a cat while following you around they will be distracted chasing it away.
+
+Rats will pick up any item stack on the floor they come across, and bring it back to their owner if they are tamed. If they are hurt and the item is edible, they will consume it and restore an amount of health equivalent to the amount of saturation the food gives as well as apply any effects the food provides. No matter their state, if rats come across a drinkable potion, they will consume it and get its effects and, once again, convert saturation to regeneration.
+
+#### World Rat Day
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
##### World Rat Day
```

File: wiki/rats-mischief.md [hunk 6]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+On April 4th, World Rat Day, all rats will smug dance and the chance of golden rats spawning increases from 1 in 150 to 1 in 30!
+
+### Types
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Types
```

File: wiki/rats-mischief.md [hunk 7]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+Rats can spawn in eight different variants and also have a few special variants that you can acquire by renaming your rats with a nametag.
+
+#### Natural Variants
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
##### Natural Variants
```

File: wiki/rats-mischief.md [hunk 8]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+- **Russian Blue**
+- **Light Brown**
+- **Chocolate**
+- **Black**
+- **Golden** (spawn chance of 1 in 150)
+
+![NaturalRatVariants](https://user-images.githubusercontent.com/83953120/117954718-95932400-b317-11eb-847f-8b326bd2f8f4.png)
+
+
+#### Special Variants
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
##### Special Variants
```

File: wiki/rats-mischief.md [hunk 9]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+- **Jerma** (Jermas Rat Movie)
+- **Biggie Cheese**
+- **Rateline** (Madeline, Celeste)
+- **Rat Kid** (with 16 variants for each minecraft dye, can be redyed just like sheep, A Hat in Time)
+
+![SpecialRatVariants](https://user-images.githubusercontent.com/83953120/117954735-9b890500-b317-11eb-8cc4-383f550f1a71.png)
+
+![RatKidVariants](https://user-images.githubusercontent.com/83953120/117954763-a04db900-b317-11eb-8ce1-02ab109df7b6.png)
+
+### Elytrats
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Elytrats
```

File: wiki/rats-mischief.md [hunk 10]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+Elytrats are a special rat variant, acquired by equipping rats with the elytrat item, crafted as shown below. Equip/unequip a rat by simply shift-using the rat with the elytrat/bare fist. Normal rats will also equip elytrats upon picking them up from the floor.
+
+![ElytratRecipe](https://user-images.githubusercontent.com/83953120/117955672-924c6800-b318-11eb-9c03-5f0d704d1f16.png)
+
+Elytrats can and will fly to attack enemies and to follow you on your airborne adventures! Just as any other rat they can be picked up with rat pouches. Elytrats also feature custom elytra textures for special rat variants and the golden rat.
+
+![Elytrat](https://user-images.githubusercontent.com/83953120/117953145-25d06980-b316-11eb-8da7-719b824ff8a8.png)
+![ElytratFormation](https://user-images.githubusercontent.com/83953120/117954803-ab084e00-b317-11eb-8a0d-07002855a6a1.png)
+
+## Accessories
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
### Accessories
```

File: wiki/rats-mischief.md [hunk 11]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+### Staves
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Staves
```

File: wiki/rats-mischief.md [hunk 12]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+
+When appearing naturally in an empty village, rats may spawn with staves in their possession. These staves can be obtained by taming the rats possessing them and are useful to make the rats perform specific tasks. Each staff use orders the rats to execute the corresponding order for 15 seconds, items acquired by the rats will be brought to you and required items can be dropped on the floor for them.
+
+- **Harvest Staff:** Harvest any mature crop and replant any cultivatable item on farmland tiles.
+- **Love Staff:** Rats breed all the animals around them (including themselves) as long as they have the corresponding food.
+- **Collection Staff:** Use with an item in the other hand to make rats break the corresponding blocks they come across. Rats can break blocks with a hardness of 1 plus 1 per level of strength that is applied to them. Haste effects boost their mining speed by 20% per level of haste.
+
+### Rat Pouches
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Rat Pouches
```

File: wiki/rats-mischief.md [hunk 13]
```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
...
+Rats stored in Pouches won't age, meaning they will keep status effects, health, their variant and names.
+
+![LeatherPouchRecipe](https://user-images.githubusercontent.com/83953120/117955697-98424900-b318-11eb-91c9-40ee74895ab7.png)
+
+![TwistedPouchRecipe](https://user-images.githubusercontent.com/83953120/117955711-9c6e6680-b318-11eb-874d-9222bd07bc94.png)
+
+![PurpurPouchRecipe](https://user-images.githubusercontent.com/83953120/117955733-a1331a80-b318-11eb-9bc9-4feb6685430b.png)
+
+
+### Mask of Rat
```
Comment (Pyrofab) [inline wiki/rats-mischief.md]:
```suggestion
#### Mask of Rat
```

**Comment by Xiribidus** [comment]
If I am not wrong, which there is a slight chance for, we do not need the assets anymore since I did the pictures _differently_, also they are compatible with dark mode now (on the other wiki, the one I did first)

**Review by Pyrofab — CHANGES_REQUESTED** [review]

**Review by Pyrofab — APPROVED** [review]
Good job, thanks !

**Pull request merged**